src/viking.js

```diff
@@ -1,11 +1,88 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength){
+        this.health = health;
+        this.strength = strength;
+        this.damage = 0;
+    }
+
+    attack(){
+        return this.strength;
+    };
+
+    receiveDamage(damage){
+        this.damage = damage;
```

Comment:
Here, `damage` is going to be the argument of your `receiveDamage()` method, so, you don't need to reference it to the `class` as you are receiving some value (later on) and that's the value you are going to use to remove it from the `health`.